src/angular-masonry.js

```diff
@@ -52,7 +52,7 @@
         $element.addClass('loaded');
       }
 
-      this.appendBrick = function appendBrick(element, id) {
+      this.appendBrick = function appendBrick(element, id, prepend) {
```

Comment:
I'm not a fan of boolean arguments. Mind splitting this up and turning this into a `prependBrick` function instead?